03-lists.md

```diff
@@ -167,15 +167,49 @@ odds: [1, 3, 5, 7]
 
 This is different from how variables worked in lesson 1, and more similar to how a spreadsheet works.
 
-## Turn a string into a list {.challenge}
+> ## Turn a string into a list {.challenge}
 
-Use a for-loop to convert the string "hello" into a list of letters:
+> Use a for-loop to convert the string "hello" into a list of letters:
 
-~~~ {.python}
-["h", "e", "l", "l", "o"]
-~~~
-Hint: You can create an empty list like this:
+> ~~~ {.python}
+> ["h", "e", "l", "l", "o"]
+> ~~~
+> Hint: You can create an empty list like this:
+
+> ~~~ {.python}
+> my_list = []
+> ~~~
+
+Subsets of lists and strings can be accessed by specifying ranges of values in brackets, similar to how we accessed ranges of positions in a Numpy matrix. This is commonly referred to as "slicing" the list/string.
 
 ~~~ {.python}
-my_list = []
+date = "Monday 4 January 2016"
+months = ["jan", "feb", "mar", "apr", "may", "jun", "jul", "aug", "sep", "oct", "nov", "dec"]
+day = date[0:6]
+year = date[17:21]
+summer = months[5:8]
+last = months[-1]
+print("day", day)
```

Comment:
The output right now was a bit confusing on first glance; I think it would help visually to add a colon in each print statement (i.e., `print("day:", day)`, without output `day: Monday`).